citi-onboarding/DADe-onepage
# feat(feature/view-contact): created contact view (without forms)

## What I did

- Created contact view and its necessary components
- Adjusted Organization text still expanding in screens bigger than 1920px (width)

| Mockup | Real |
|--------|------|
| ![contact mockup real](https://user-images.githubusercontent.com/61971624/122241522-fffc2e80-ce98-11eb-843b-ba73f59ff43c.png) | ![contact mockup made](https://user-images.githubusercontent.com/61971624/122282053-5d0bda80-cec1-11eb-859b-41435c5085a9.png)|


## How to test


1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/view-contact`
   4. `git pull origin feature/view-contact`
   5. `npm install`
   6. `cd client/`
   7. `npm start`

3. In your browser go to http://localhost:3000 (should open automatically when running `npm start`)

4. Check if the component (Contact) looks just like the mockup in different screen sizes (minimum width: 1366px) and if when you increase it past 1920px it remains the same size (Doesnt grow along with the screen).

5. In VScode open the `client\src\Views\Contact\Contact.jsx` file

6. In lines 37 to 41 of the mentioned file, change the contactLink property to any link, save the file, and check if when you click in the social media name it takes you to this new link.

7. Don't commit or push your changes from the tests, as the original informations were the correct ones for the current version of the project. 

8. Test the things that you think are worth testing, even the ones that are not in this description
